app/policies/tournament_policy.rb

```diff
@@ -5,6 +5,7 @@ def resolve
       scope.all
     end
 
+    # TODO: Should this be in scope?
```

Comment:
I believe this is not actually necessary, it is basically redundant with the scope.all thing